InfiniteRecharge/src/main/java/frc/robot/subsystem/climber/ClimbSubsystem.java

```diff
@@ -79,6 +79,18 @@ public boolean isActive(){
         return active;
     }
 
+    public void moveArms(double leftStick, double rightStick) {
+        if (!isActive()) {
+            return;
+        }
+
+        if (!isRewindEnabled()) {
+            pitRewind(leftStick, rightStick);
+        } else if (isRewindEnabled()) {
+            manualClimb(leftStick, rightStick);
+        }

```

Comment:
This looks backwards. I think if rewind is enabled, we want it to do pitRewind, otherwise we want it to do manulClimb.